validations/users.js

```diff
@@ -7,24 +10,40 @@ const userRegValidators = [
         .isLength({ max: 50 })
         .withMessage('User Name must not be more than 50 characters long')
         .matches(/@/)  
-        .withMessage('User Name cannot contain @ symbol'),
+        .withMessage('User Name cannot contain @ symbol')
+        .custom(async (value) => {
+            const username = await User.findOne({where: {username: value}})
+            if(!username) {

```

Comment:
No bang goes here. You want to throw the error if a user with username WAS found